scripts/pi-hole/php/get.php

```diff
@@ -12,54 +12,42 @@
 
 $listtype = $_GET['list'];
 
+$basedir = "/etc/pihole/";
+
 require "func.php";
 
 switch ($listtype) {
-	case "white":
-		$list = getListContent("white");
-		break;
-
-	case "black":
-		$list = array_merge(getListContent("black"),getWildcardListContent());
-		break;
-
-	default:
-		die("Invalid list parameter");
-		break;
+    case "white":
+        $list = array(getListContent("whitelist.txt"));
+        break;
+
+    case "black":
+        $exact = array(getListContent("blacklist.txt"));
+        $regex = array(getListContent("regex.list"));
+        $list = array_merge($exact, $regex);
```

Comment:
It might make more sense to do this:
```
$exact = getListContent("blacklist.txt");
$regex = getListContent("regex.list");
$list = array($exact, $regex);
```